rust/cloud-storage/authentication_service/src/main.rs

```diff
@@ -114,6 +114,15 @@ async fn main() -> anyhow::Result<()> {
             .to_string(),
     };
 
+    let stripe_price_id = match config.environment {
+        Environment::Local => config.stripe_price_id.clone(),
+        _ => secretsmanager_client
+            .get_secret_value(&config.stripe_price_id)
+            .await
+            .context("unable to get stripe price id")?
+            .to_string(),
+    };
```

Comment:
We could probably use the localorremote secret thing here, its a lot less verbose